src/file_drop/linux.rs

```diff
@@ -0,0 +1,50 @@
+use super::{FileDropEvent, FileDropHandler, FileDropListener};
+
+use std::{path::PathBuf, rc::Rc};
+
+use webkit2gtk::WebView;
+use gtk::WidgetExt;
+
+pub(crate) struct FileDropController;
```

Comment:
Are you sure? That would make it inconsistent with macOS and Windows.

Reply:
All platforms are already pretty disjointed. I think this is because I prefer just to dealing with their problems and nothing else. It's still okay you left as is.

Reply:
Alright, I will leave it as is.